commune_app/migrations/0002_alter_user_groups_alter_user_is_active.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.1.3 on 2023-01-14 12:45
+
```

Comment:
since you removed the founder, maybe this migration file is not needed anymore too